src/components/core/DpUpload/DpUpload.vue

```diff
@@ -72,6 +78,15 @@ export default {
       type: Object,
       required: false,
       default: () => ({})
+    },
+
+    /**
+     * Global path for file uploader endpoint.
+     */
+    tusEndpoint: {
+      type: String,
+      required: true,
+      default: ''
```

Comment:
Required props do not need a default value.

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/516/commits/059756c2ff2d7fc3265dcb18979681b27367645b